draft-ietf-dnsop-serve-stale.md

```diff
@@ -160,7 +151,7 @@ time to live."  This is again not {{RFC2119}}-normative language, but
 does convey the natural language connotation that data becomes
 unusable past TTL expiry.
 
-Several major recursive resolver operators currently use stale data
+Several recursive resolver packages and their operators currently use stale data
```

Comment:
Wording is a little awkward since operators don't operate the packages, they operator a service.
Suggest: "Several recursive resolver operators currently use stale data for answers in some way, including Akamai. A number of recursive resolver packages (including BIND, Know, OpenDNS, Unbound) provide options to use stale data."

Reply:
Puneet's proposed wording is definitely better than mine.